pyproject.toml

```diff
@@ -1,6 +1,83 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+  "hatch-fancy-pypi-readme",
+  "hatch-vcs",
+  "hatchling",
+]
+
+[project]
+name = "django-simple-history"
+description = "Store model history and view/revert changes from admin site."
+maintainers = [
+  { name = "Trey Hunner" },
+]
+authors = [
+  { name = "Corey Bertram", email = "corey@qr7.com" },
+]
+requires-python = ">=3.8"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Framework :: Django",
+  "Framework :: Django :: 4.2",
+  "Framework :: Django :: 5.0",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  # DEV: uncomment this when the `pyproject-fmt` pre-commit hook stops removing it
+  #"Programming Language :: Python :: 3.13",
+]
+dynamic = [
+  "readme",
+  "version",
+]
+dependencies = [
+]
+urls.Changelog = "https://github.com/jazzband/django-simple-history/blob/master/CHANGES.rst"
+urls.Documentation = "https://django-simple-history.readthedocs.io/en/stable/"
+urls.Homepage = "https://github.com/jazzband/django-simple-history"
+urls.Source = "https://github.com/jazzband/django-simple-history"
+urls.Tracker = "https://github.com/jazzband/django-simple-history/issues"
+
+[tool.hatch.version]
+source = "vcs"
+fallback-version = "0.0.0"
+
+[tool.hatch.version.raw-options]
+version_scheme = "no-guess-dev"
+local_scheme = "node-and-date"
+
+[tool.hatch.build.targets.wheel]
+# Jazzband's release process is limited to 2.2 metadata
```

Comment:
We should link to the roadie issue Hugo has to make it easier to know when this can be removed. The same for the other case too.

Reply:
Good idea! 👍